lxd/storage/drivers/driver_pure.go

```diff
@@ -76,6 +102,11 @@ func (d *pure) Info() Info {
 
 // FillConfig populates the storage pool's configuration file with the default values.
 func (d *pure) FillConfig() error {
+	// Use iSCSI by default.
+	if d.config["pure.mode"] == "" {
+		d.config["pure.mode"] = pureModeISCSI
```

Comment:
should we prefer nvme by default like powerflex?

Reply:
Fixed - it was set to NVMe by default, but in the later commit.